pygpu/gpuarray.pyx

```diff
@@ -510,6 +510,8 @@ def init(dev, sched='default', disable_alloc_cache=False):
     :type sched: {'default', 'single', 'multi'}
     :param disable_alloc_cache: disable allocation cache (if any)
     :type disable_alloc_cache: bool
+    :param single_stream: enable single stream mode
```

Comment:
Yes that is exactly it.  The flags determines the synchronization mechanism to use. One of them is heavy on the CPU, but fast.  The other will yield, but introduces additional delays when synchronizing.